.github/workflows/main.yml

```diff
@@ -172,83 +167,3 @@ jobs:
         run: |
           pytest tests/test_main_system_caches.py
           pytest -m 'vetiver'
-
-  test-connect:
-    needs: distributions
-    runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        PY_VERSION:
-          - 3.11.13
-          - 3.12.11
-    steps:
-          - uses: extractions/setup-just@v2
-            env:
-              GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          - uses: actions/checkout@v4
-            with:
-              fetch-depth: 0
-            env:
-              GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          # Checkout the rsconnect-python tests from Connect
-          - uses: actions/checkout@v4
-            with:
-              repository: rstudio/connect
-              path: 'test/connect-rsconnect-python'
-              sparse-checkout: |
-                  test/rsconnect-python
-                  tools/dev
-                  examples
-              sparse-checkout-cone-mode: false
-              token: ${{ secrets.CONNECT_PAT }}
-
-          - name: Delete dotnet to save space
-            run: sudo rm -rf /usr/share/dotnet
-
-          - name: Build docker container-image
-            run: |
-              cd test/connect-rsconnect-python/test/rsconnect-python/
-              docker compose --profile rsconnect build
-
-          - name: Restore dist
-            uses: actions/download-artifact@v4
-            with:
-              name: distributions
-              path: dist/
-
-          - name: Run rsconnect-python Tests
-            env:
-              CONNECT_LICENSE: "${{ secrets.RSC_LICENSE }}"
-              PY_VERSION: ${{ matrix.PY_VERSION }}
-              TEST_SUBSET: "CI"
-              RSC_AUTOMATION_PAT: "${{ secrets.CONNECT_PAT }}"
-              ADMIN_API_KEY: "${{ secrets.ADMIN_API_KEY }}"
-              QUARTO_VERSION: "1.4.546"
-
-              # This allows us to start Connect separately in our own docker container
-              CONNECT_SERVER: "http://localhost:3939"
-              remote: "yes"
-            run: |
-              cd integration-testing
-              docker compose pull connect
-              docker compose up -d connect
-              just ../test/connect-rsconnect-python/test/rsconnect-python/test-rsconnect-python-repo
```

Comment:
Once this merges, we can tidy up this justfile in connect and at least remove this target, which nothing else calls. Probably worth doing an audit there of any other stuff, like in the `docker compose` setup, to see if there's more that we can prune.

Reply:
Connect ticket for this follow up work: https://github.com/posit-dev/connect/issues/34626